testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/InsertTest.java

```diff
@@ -69,6 +70,21 @@ public void testSimpleInsert() {
     testSimpleInsert("insertUser");
   }
 
+  @Test
+  @DisplayName("Should map simple insert return boolean")
+  public void testSimpleInsertReturnBoolean() {
+    Object response =
+        CLIENT.executeKeyspaceQuery(
+            KEYSPACE,
+            "mutation {\n"
+                + "  result: insertUserReturnBoolean(user: {name: \"Ada Lovelace\", username: \"@ada\"})\n"
+                + "}");
+
+    // Should have generated an id
+    Boolean applied = JsonPath.<Boolean>read(response, "$.result");
```

Comment:
Awesome, I like this `JsonPath` helper, way more easier to assert the results. We should use this everywhere..